Loop/Managers/Alerts/DeviceAlertManager.swift

```diff
@@ -44,18 +44,22 @@ public final class DeviceAlertManager {
     
     let userNotificationCenter: UserNotificationCenter
     let fileManager: FileManager
+    
+    let alertStore: AlertStore
 
     public init(rootViewController: UIViewController,
                 handlers: [DeviceAlertPresenter]? = nil,
                 userNotificationCenter: UserNotificationCenter = UNUserNotificationCenter.current(),
                 fileManager: FileManager = FileManager.default) {
         self.userNotificationCenter = userNotificationCenter
         self.fileManager = fileManager
+        let documentsDirectory = fileManager.urls(for: .documentDirectory, in: .userDomainMask).first!
+        alertStore = AlertStore(storageFileURL: documentsDirectory.appendingPathComponent("AlertStore.sqlite"))
```

Comment:
Might want to put this into it's own directory, to contain and allow easy cleanup of the group of files that this creates (the write ahead log, shm file, etc.)

Reply:
Good idea!  I'll just use a subdirectory called `AlertStore`, unless you want to stuff other storage in that directory too...